Index: dtee-1.1.1/docs/meson.build
===================================================================
--- dtee-1.1.1.orig/docs/meson.build
+++ dtee-1.1.1/docs/meson.build
@@ -39,5 +39,10 @@ if sphinx_build.found()
 		install: true,
 		install_dir: man_dir)
 
-	install_symlink('cronty.1', pointing_to: 'dtee.1', install_dir: man_dir)
+	custom_target('cronty.1',
+		input: dtee_man,
+		output: 'cronty.1',
+		command: [ln, '-s', '-f', '@PLAINNAME@', '@OUTPUT@'],
+		install: true,
+		install_dir: man_dir)
 endif
Index: dtee-1.1.1/meson.build
===================================================================
--- dtee-1.1.1.orig/meson.build
+++ dtee-1.1.1/meson.build
@@ -8,7 +8,7 @@ project('dtee',
 	],
 	version: '1.1.1',
 	license: 'GPL3+',
-	meson_version: '>=0.63.0',
+	meson_version: '>=0.56.2',
 )
 
 c = meson.get_compiler('c')
@@ -90,7 +90,14 @@ dtee = executable('dtee',
 	install_rpath: rpath,
 	install: true)
 
-install_symlink('cronty', pointing_to: 'dtee', install_dir: get_option('bindir'))
+ln = find_program('ln')
+
+custom_target('cronty',
+	input: dtee,
+	output: 'cronty',
+	command: [ln, '-s', '-f', '@INPUT@', '@OUTPUT@'],
+	install: true,
+	install_dir: get_option('bindir'))
 
 subdir('docs')
 subdir('tests')
Index: dtee-1.1.1/tests/util/common.sh
===================================================================
--- dtee-1.1.1.orig/tests/util/common.sh
+++ dtee-1.1.1/tests/util/common.sh
@@ -18,7 +18,7 @@ while [ "$(basename -- "$dir")" != "test
 done
 
 ln -s ../dtee tests/dtee
-ln -s dtee tests/cronty
+ln -s ../cronty tests/cronty
 BASETESTDIR="dtee@test"
 TESTDIR="$BASETESTDIR"
 if [ "$GROUP" != "" ]; then
